__tests__/gui/common_cy_functions/general.js

```diff
@@ -139,11 +139,21 @@ export function clickOnTheFirstCategory() {
     cy.get('.point-mapper .point-mapper-content__list .point-mapper__h1 .point-mapper__h1_content .point-mapper__h2_wrapper .point-mapper__h2').first().click();
 }
 
-export function hoverOverTheMapCenter() {
+export function hoverOverTheMapCenter(elementSelectorToFind = null, retryCount = 5) {
     const coordinates = getMapCenter();
     cy.get('body')
         .trigger('mousemove', {clientX: 0, clientY: 0})
         .trigger('mousemove', {clientX: coordinates.x, clientY: coordinates.y});
+
+    if (elementSelectorToFind !== null) {
+        cy.get("body").then(($body) => {
+            cy.log($body.find(elementSelectorToFind).length)
+            if (!$body.find(elementSelectorToFind).length && retryCount > 1) {
+                cy.log('hover failed - try again');
```

Comment:
This log is could be useful for debugging failed CI, so it can stay
If you want you can include the retryCount like
```
cy.log(`hover failed - trying %(retryCount - 1) more`);
```